compiler/testData/debug/localVariables/constructors/multipleConstructors.kt

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
As mentioned in the PR, these stepping tests are not directly related to this PR, but support the corresponding PR on the plug-in side by clarifying the stepping behavior of the JVM backend v/ the IR backend.